docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 
 services:
+    grobid-base:
+        build:
+            context: .
+            dockerfile: Dockerfile.base
+        image: elifesciences/sciencebeam-grobid-biorxiv_base:${IMAGE_TAG}
+        command: /bin/sh -c exit 0
     grobid:
         build:
             context: .
             dockerfile: ${GROBID_DOCKERFILE:-Dockerfile}
```

Comment:
Default `Dockerfile` not part of your code, so, are you planning to add one to the code?

Reply:
No, I will remove the default.